### PR TITLE
deew: Add version 3.0.1

### PR DESCRIPTION
**Pull request by semisoft0072**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/deew.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "3.0.1",
+    "description": "Dolby Encoding Engine Wrapper",
+    "homepage": "https://github.com/pcroland/deew",
+    "license": "MIT License",
+    "suggest": {
+        "ffmpeg": [
+            "ffmpeg",
+            "versions/ffmpeg-nightly"
+        ]
```
Comment (HUMORCE) [inline bucket/deew.json:10]:
```suggestion
        "ffmpeg": "ffmpeg"
```

Suggests stable package

File: bucket/deew.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "license": "MIT License",
+    "suggest": {
+        "ffmpeg": [
+            "ffmpeg",
+            "versions/ffmpeg-nightly"
+        ]
+    },
+    "url": "https://github.com/pcroland/deew/releases/download/3.0.1/deew.exe",
+    "hash": "c621b9af9b037f78be4626c5b592dbf04311d214bc7ee46c0808eddd4f9d5ca3",
+    "bin": "deew.exe",
```
Comment (HUMORCE) [inline bucket/deew.json:14]: `deew.exe` is a 64bit program, use `architecture`.

File: bucket/deew.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "checkver": {
+        "github": "https://github.com/pcroland/deew"
+    },
+    "autoupdate": {
+        "url": "https://github.com/pcroland/deew/releases/download/$version/deew.exe"
+    }
```
Comment (HUMORCE) [inline bucket/deew.json:20]: update autoupdate in same time.

File: bucket/deew.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "3.0.1",
+    "description": "Dolby Encoding Engine Wrapper",
+    "homepage": "https://github.com/pcroland/deew",
+    "license": "MIT License",
```
Comment (HUMORCE) [inline bucket/deew.json:5]:
```suggestion
    "license": "MIT",
```
Use SPDX License identifier.

**Review by HUMORCE — CHANGES_REQUESTED** [review]

**Review by semisoft0072 — COMMENTED** [review]
added architecture and autoupdate in same time

File: bucket/deew.json (hunk 2 above)
Comment (semisoft0072) [reply bucket/deew.json]: added

**Review by semisoft0072 — COMMENTED** [review]
deew depends on FFmpeg

**Review by HUMORCE — APPROVED** [review]

**Comment by HUMORCE** [comment]
> deew depends on FFmpeg

For ffmpeg, I prefer to handle it [this method](https://github.com/ScoopInstaller/Main/blob/master/bucket/android-clt.json#L28-L30), then that users can choose the build they want to install.

```
'main' bucket:
  ffmpeg-shared (6.0)
  ffmpeg (6.0) [installed]

'versions' bucket:
  ffmpeg-gyan-nightly (2023-04-26)
  ffmpeg-nightly (1682603237)
  ffmpeg-shared-nightly (110363)
  imagemagick-ffmpeg (7.1.0-53)
```

**Comment by semisoft0072** [comment]
@HUMORCE Sorry i did not get it which method you talking about,
But for ffmpeg selection i did this last time https://github.com/semisoft0072/scoop-apps/blob/7557dd4e55b65965d603c11847ea99b1b0944e33/bucket/deew.json#L12-L37

**Comment by semisoft0072** [comment]
@HUMORCE got it please verify the changes whenever you feel free

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**